sdk/sdk-skeleton.go

```diff
@@ -0,0 +1,126 @@
+import "time"
+
+type Asset struct{}
+
+//// Methods directly on the SDK
+
+func NewChannel() {
+
+}
+
+func NewChannelListenerService() {
+
+}
+
+func GetChannel() {
+
+}
+
+//// Channel State Machine
+
+type ChannelStateMachine struct {
+	ChannelID              string
+	Status                 string // waiting | active | closing | closed
+	Asset                  Asset
+	InitiatorEscrowAccount string
+	ResponderClaimAmount   int
+	ReserveEscrowAccount   string
```

Comment:
The SEP doesn't have a reserve account any longer, but two mostly equal escrow accounts. The only thing that differs between the two is which escrow account is used for sequencing.

Reply:
right, I think ResponderEscrowAccount is more consistent with the docs
changed: https://github.com/stellar/experimental-payment-channels/pull/25/commits/bbbff035f84586fa306758c6eb2d02c9ecc4719b